src/recensio/plone/utils.py

```diff
@@ -72,8 +72,7 @@ def formatter(*args):
     return formatter
 
 
-def get_formatted_names(names, lastname_first=False):
-    full_name_separator = " / "
+def get_formatted_names(names, lastname_first=False, full_name_separator=" / "):
```

Comment:
Is this breaking the code where this utility was already used?

Reply:
No, but see also https://github.com/syslabcom/recensio.plone/pull/59

Reply:
Uhm, we should avoid rewriting code signatures while migrating the code.
If it would have been copied immediately as implemented in the old system this discussion could have been avoided.